formats/email_alert_signup/publisher/details.json

```diff
@@ -13,17 +13,7 @@
       "minProperties": 1,
       "properties": {
         "tags": {
-          "type": "object",
-          "description": "DEPRECATED: The tags used to match subscribers lists",
```

Comment:
On closer inspection, this description was replaced with the following, which I think is clearer:
> Field used by email-alert-api to trigger email alerts for subscriptions to topics (gov.uk/topic) and policies (gov.uk/policies).